launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
```

Comment:
Mainly, I wanted to make it safe to require this package without actually launching anything, so that (for example), someone could safely require the router in their own work.  I do see the convenience of being able to use the `node .`, but we do need to support derivative works a bit better.

I'd suggest moving the existing `index.js` to something like `require.js` and then setting the main option in `package.json`.  We can then move `launcher.js` to `index.js`, and the previous syntax will work.